src/main/groovy/com/lesfurets/jenkins/unit/PipelineTestHelper.groovy

```diff
@@ -1,5 +1,7 @@
 package com.lesfurets.jenkins.unit
 
+import com.sun.org.apache.xpath.internal.operations.Bool
```

Comment:
Hello @nre-ableton !
Why this unused import com.sun.org.apache.xpath.internal.operations.Bool here?